rockspec/apisix-dev-0.rockspec

```diff
@@ -22,6 +22,7 @@ dependencies = {
    "lua-resty-ngxvar = 0.3",
    "lua-resty-jit-uuid = 0.0.7",
    "rapidjson = 0.6.0-1",
+   "lua-resty-healthcheck-iresty = 1.0.0",
```

Comment:
we should add the copyright of lua-resty-event and lua-resty-healthcheck.

Reply:
I have added todo items before.

![image](https://user-images.githubusercontent.com/6814606/61303008-5a0d9880-a819-11e9-9f7c-dbc1de23dc8c.png)